lib/types/index.js

```diff
@@ -385,6 +384,7 @@ export type Feed = {
   id: string,
   isCreating?: boolean,
   isPublic: boolean,
+  labels: Array<Label>,
```

Comment:
Please remove if refactoring data model as recommended in https://github.com/ibi-group/datatools-server/pull/403#pullrequestreview-714190079.